apache/collabosphere.template

```diff
@@ -55,8 +55,8 @@
         ExpiresDefault "access plus 1 year"
     </LocationMatch>
 
-    # Don't cache the root HTML pages or the bookmarklet init script
-    <LocationMatch "^(/assets/js/bookmarklet-init.js|/bookmarklet.html)">
+    # Don't cache the root HTML pages or public scripts
+    <LocationMatch "^(/assets/js/|/bookmarklet.html)">
```

Comment:
@johncrossman This is intended to match anything under `/assets/js/` - because there's no `$` at the end, it will match just the start of the path.